1-js/05-data-types/02-number/article.md

```diff
@@ -271,13 +274,11 @@ JavaScript doesn't trigger an error in such events. It does its best to fit the
 ```smart header="Two zeroes"
 Another funny consequence of the internal representation of numbers is the existence of two zeroes: `0` and `-0`.
 
-That's because a sign is represented by a single bit, so every number can be positive or negative, including a zero.
+That's because a sign is represented by a single bit, so it can be set or not set for any number including a zero.
```

Comment:
I prefer the original. Or, to get more wordy: "...a single bit which can be set for any number, so every number can be positive or negative, including zero". But I find it pretty clear the way it is (or was).

To me "set" does not necessarily mean "set to '1'" -- it could also be set to '0'. So I don't see "set or not set" as being necessary.